packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/models/index.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
same thought here, maybe we don't need to unify the models for multiple clients into one top level models export?

Reply:
The story with models is a little different, I am totally okay to remove the subpath of @azure/load-testing/models, but if the intention of removing it is to avoid conflicts, we will probably not get this benefit even remove it, because we still export those models in the src/index.ts https://github.com/Azure/autorest.typescript/pull/1830/files#diff-84b4020119a80f9e0d214073a464a463e61cdc9802e29f5f0b00b7bbc37880e5R69